test/core/return.wast

```diff
@@ -320,14 +320,6 @@
   "type mismatch"
 )
 
-(assert_invalid
-  (module
-    (func $type-operand-missing (result i32)
-      (return)
-    )
-  )
-  "type mismatch"
-)
 (assert_invalid
   (module
     (func $type-operand-missing-in-block (result i32)
```

Comment:
Can we rename these to $type-value-empty-vs-num-in-*` and move them next to the basic version of the test?

Reply:
Fine. :)